### PR TITLE
update sample app

### PR DESCRIPTION
**Pull request by paul-turner**
On behalf of the wonderful @tir38: 

This will update the sample app by using a realistic example: recipes and ingredients. By having just numbered parents and children, it wasn't clear what was XRV code and what was implementation code.

sample

### CODE REVIEW

File: sampleapp/src/main/java/com/ryanbrooks/expandablerecyclerviewsample/linear/vertical/VerticalLinearRecyclerViewSampleActivity.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        final List<Recipe> recipes = Arrays.asList(taco, quesadilla);
+
+        RecyclerView recyclerView = (RecyclerView) findViewById(R.id.recyclerview);
+        mAdapter = new RecipeAdapter(this, recipes);
+        mAdapter.setExpandCollapseListener(new ExpandableRecyclerAdapter.ExpandCollapseListener() {
+            @Override
+            public void onListItemExpanded(int position) {
+                Recipe expandedRecipe = recipes.get(position);
+                Toast.makeText(VerticalLinearRecyclerViewSampleActivity.this,
+                        "expanded: " + expandedRecipe.getName(),
```
Comment (dgreenhalgh) [inline sampleapp/src/main/java/com/ryanbrooks/expandablerecyclerviewsample/linear/vertical/VerticalLinearRecyclerViewSampleActivity.java]: Should be pulled into strings.xml

File: sampleapp/src/main/java/com/ryanbrooks/expandablerecyclerviewsample/linear/vertical/VerticalLinearRecyclerViewSampleActivity.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                        Toast.LENGTH_SHORT)
+                        .show();
+            }
+
+            @Override
+            public void onListItemCollapsed(int position) {
+                Recipe collapsedRecipe = recipes.get(position);
+                Toast.makeText(VerticalLinearRecyclerViewSampleActivity.this,
+                        "collapsed: " + collapsedRecipe.getName(),
```
Comment (dgreenhalgh) [inline sampleapp/src/main/java/com/ryanbrooks/expandablerecyclerviewsample/linear/vertical/VerticalLinearRecyclerViewSampleActivity.java]: Should be pulled into strings.xml

File: sampleapp/src/main/res/layout/ingredient_view.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:orientation="vertical"
+              android:padding="16dp">
```
Comment (dgreenhalgh) [inline sampleapp/src/main/res/layout/ingredient_view.xml]: Should be pulled into dimens.xml

File: sampleapp/src/main/res/layout/ingredient_view.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+    <TextView android:id="@+id/ingredient_textview"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:textSize="16sp"/>
```
Comment (dgreenhalgh) [inline sampleapp/src/main/res/layout/ingredient_view.xml]: Should be pulled into dimens.xml

File: sampleapp/src/main/res/layout/recipe_view.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:orientation="horizontal"
+              android:padding="16dp">
```
Comment (dgreenhalgh) [inline sampleapp/src/main/res/layout/recipe_view.xml]: Should be pulled into dimens.xml

File: sampleapp/src/main/res/layout/recipe_view.xml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+    <TextView android:id="@+id/recipe_textview"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:layout_weight="1"
+              android:textSize="16sp"
```
Comment (dgreenhalgh) [inline sampleapp/src/main/res/layout/recipe_view.xml]: Should be pulled into dimens.xml

**Comment by dgreenhalgh** [comment]
This looks good outside of those minor values comments. We also may want to add some more recipes so that we have enough to scroll on an average device oriented vertically.

**Comment by tir38** [comment]
Cool I'll take care if this today.

On Wed, Jan 6, 2016, 9:51 AM David Greenhalgh <notifications@github.com>
wrote:

> This looks good outside of those minor values comments. We also may want
> to add some more recipes so that we have enough to scroll on an average
> device oriented vertically.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bignerdranch/expandable-recycler-view/pull/142#issuecomment-169343596>
> .
>
-- 
Jason Atwood
Android Developer and Instructor
Big Nerd Ranch

File: sampleapp/src/main/java/com/ryanbrooks/expandablerecyclerviewsample/linear/vertical/VerticalLinearRecyclerViewSampleActivity.java (hunk 2 above)
Comment (tir38) [inline sampleapp/src/main/java/com/ryanbrooks/expandablerecyclerviewsample/linear/vertical/VerticalLinearRecyclerViewSampleActivity.java]: :heavy_check_mark:

File: sampleapp/src/main/java/com/ryanbrooks/expandablerecyclerviewsample/linear/vertical/VerticalLinearRecyclerViewSampleActivity.java (hunk 1 above)
Comment (tir38) [inline sampleapp/src/main/java/com/ryanbrooks/expandablerecyclerviewsample/linear/vertical/VerticalLinearRecyclerViewSampleActivity.java]: :heavy_check_mark:

File: sampleapp/src/main/res/layout/ingredient_view.xml (hunk 3 above)
Comment (tir38) [inline sampleapp/src/main/res/layout/ingredient_view.xml]: :heavy_check_mark:

File: sampleapp/src/main/res/layout/recipe_view.xml (hunk 5 above)
Comment (tir38) [inline sampleapp/src/main/res/layout/recipe_view.xml]: :heavy_check_mark:

File: sampleapp/src/main/res/layout/recipe_view.xml (hunk 6 above)
Comment (tir38) [inline sampleapp/src/main/res/layout/recipe_view.xml]: :heavy_check_mark:

File: sampleapp/src/main/res/layout/ingredient_view.xml (hunk 4 above)
Comment (tir38) [inline sampleapp/src/main/res/layout/ingredient_view.xml]: :heavy_check_mark:

**Comment by dgreenhalgh** [comment]
Looks good :+1:

**Pull request merged**